chapter_natural-language-processing-pretraining/bert-pretraining.md

```diff
@@ -175,6 +173,7 @@ def train_bert(train_iter, net, loss, vocab_size, devices, num_steps):
 ```{.python .input}
 #@tab pytorch
 def train_bert(train_iter, net, loss, vocab_size, devices, num_steps):
+    net(*next(iter(train_iter))[:4]) # A dry run to initialize the model.
```

Comment:
Keep 2 space right before an inline comment. Remove period at the end of the comment.

Reply:
It is resolved, thanks!